frontend/src/main/pages/ProfilePage.js

```diff
@@ -34,9 +33,7 @@ const ProfilePage = () => {
                     <RoleBadge role={"ROLE_ADMIN"} currentUser={currentUser}/>
                 </Col>
             </Row>
-            <Row className="text-left">
-                <ReactJson src={currentUser.root} />
-            </Row>
+            
```

Comment:
Since we are removing the JSON display, can we go ahead and add code here to display the roles that are appropriate to this app, i.e. 

```suggestion
            <RoleBadge role={"ROLE_DRIVER"} currentUser={currentUser}/>
             <RoleBadge role={"ROLE_RIDER"} currentUser={currentUser}/>
```

Note that this suggestion will put it in the wrong place; it should go next to the other `RoleBadge` items.